### PR TITLE
More lints

### PR DESCRIPTION
**Pull request by natebosch**

### CODE REVIEW

File: analysis_options.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ analyzer:
   strong-mode:
     implicit-casts: false
   errors:
+    todo: ignore
```
Comment (kevmoo) [inline analysis_options.yaml:6]: does `todo` come up?

**Review by kevmoo — APPROVED** [review]
I'm on the fence about cascades, but if you like it...

File: analysis_options.yaml (hunk 1 above)
Comment (natebosch) [reply analysis_options.yaml:6]: it hasn't yet in this package but I'm in the habit of adding this line anytime I'm doing a bigger cleanup.

**Comment by natebosch** [comment]
> I'm on the fence about cascades, but if you like it...

Yeah, some of the cases for this lint I like and some I could go either way. We're trying it in the build repo and I copied the list of lints over from there. I figured I may as well keep it until we find an egregious case or learn that its too annoying day to day.

**Pull request merged**